src/runtime/virtcontainers/physical_endpoint.go

```diff
@@ -100,6 +100,7 @@ func (endpoint *PhysicalEndpoint) Attach(ctx context.Context, s *Sandbox) error
 		Major:         c.Major,
 		Minor:         c.Minor,
 		ColdPlug:      true,
+		Port:          s.config.HypervisorConfig.ColdPlugVFIO,
```

Comment:
Are physical_endpoints only cold-plugged aka direct-attached? Can those also be hot-plugged?

Reply:
The physical endpoints are the attached to a bridge or root-port? Is the config true for all architectures?

Reply:
> Are physical_endpoints only cold-plugged aka direct-attached? Can those also be hot-plugged?

Only cold-plugged here because of `ColdPlug: true`.

Reply:
> The physical endpoints are the attached to a bridge or root-port? Is the config correct for all architectures?

The config here is reading from config file which is `configuration.toml`.
There will not be the `Port` option if we can cover all cases.

Reply:
I had tested the code changes in my production environment, it works well.